cmd/crc/cmd/start.go

```diff
@@ -113,7 +117,7 @@ func getPullSecretFileContent() (string, error) {
 
 	// In case user doesn't provide a file in start command or in config then ask for it.
 	if crcConfig.GetString(config.PullSecretFile.Name) == "" {
-		pullsecret, err = input.PromptUserForSecret("Image pull secret", fmt.Sprintf("Copy it from %s", constants.DefaultPullSecretURL))
+		pullsecret, err = input.PromptUserForSecret("Image pull secret", fmt.Sprintf("Copy it from %s", constants.CrcLandingPageURL))
```

Comment:
this is related to another PR I believe. this is not about the release info

Reply:
The URL we want to point people at to find new releases is `constants.DefaultPullSecretURL` (ie same place as where the pull secret is). Since we are using it for more than "URL where the pull secret is" in this patch series, the renaming makes sense here. The renaming could be done in a separate preliminary commit, but so far we did not push much for such fine grained commits.